database/schema.sql

```diff
@@ -42,7 +42,8 @@ CREATE TABLE certificates(
     excluded_dns_domains        varchar[] NOT NULL DEFAULT '{}',
     excluded_ip_addresses       varchar[] NOT NULL DEFAULT '{}',
     is_technically_constrained  bool NOT NULL DEFAULT false,
-    cisco_umbrella_rank         integer NOT NULL DEFAULT 2147483647
+    cisco_umbrella_rank         integer NOT NULL DEFAULT 2147483647,
+    revoked_via_crl             bool NULL
```

Comment:
we already have `is_revoked bool` and `revoked_at timestamp` in the certificate table, could those be used instead of this new field?
I wonder if we need a `revocation_reason varchar` column to add extra information, like "listed in CRL XYZ and OCSP from ABC".